.github/workflows/code_sample_checker.yml

```diff
@@ -0,0 +1,34 @@
+name: Code sample checker
+on: pull_request
+
+jobs:
+    test-code-samples:
+        name: check-code-samples
+        runs-on: ${{matrix.os}}
+
+        strategy:
+            matrix:
+                os: [ ubuntu-latest, windows-latest ]
```

Comment:
Do we really need to test this on Windows too? I mean, for a code sample, it shouldn't really matter

Reply:
Does it hurt? I think it may be useful to check that code samples work on windows which is important.

Reply:
It hurts me (I care for the poor Github) but I can live with that :smile:

Reply:
Don't worry this one does not take long :)